### ISSUE TITLE
FacebookMessengerShareButton throws "Warning: React does not recognize the `appId` prop on a DOM element."

### ISSUE THREAD

**Issue by iwakoscott** [nygardk/react-share#286]
It looks like the `forwardRef` is forwarding the prop - `appId` down to the native button element. Any idea how to fix this? It works how it is but I want to get the warning off of the console. 😅 Thanks!

Here is the full warning in the console:
```
Warning: React does not recognize the `appId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `appid` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

I am running the latest version of `react-share` 4.0.1.

**Comment by developersHomefriend** [comment]
same issue here

**Comment by nygardk** [comment]
Fixed in v4.1.0
